include/uk/plat/bootstrap.h

```diff
@@ -50,19 +50,7 @@ extern "C" {
  * @param argc Number of arguments
```

Comment:
Hmm, maybe also delete the `@param` comments here?

Reply:
Ah yes, but this goes away altogether in a later commit in this PR.